test(5000-tables): change seed for 5000 tables test

soyacz opened the pull request:
We ran into problematic nemesis for 5000 tables test: https://github.com/scylladb/scylla-manager/issues/3237

This commit switch nemesis seed to make this test with different nemesis set.

## PR pre-checks (self review)
<!--- PR should be created as Draft, when CI finished and relevant checkboxes selected, add reviewers and then click on "Ready for review" button.-->
<!--- Put an `x` in all the boxes that apply or create PR and then click on all relevant checkboxes: -->
- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

fgelcer reviewed (approved):
LGTM

fruch commented on test-cases/scale/longevity-5000-tables.yaml:
```diff
@@ -41,7 +41,7 @@ user_prefix: 'longevity-5000-tables'
 
 nemesis_class_name: 'SisyphusMonkey'
-nemesis_seed: '004'
+nemesis_seed: '404'
```
it's NotFound  :)

fruch reviewed (approved):
LGTM

fruch commented:
@soyacz why sending this to master ? (and then we'll backport, like any other issue ?)

fgelcer commented:
> @soyacz why sending this to master ? (and then we'll backport, like any other issue ?)

this must be decided, but this is only a workaround for an issue with the manager... so, hopefully it will be fixed soon, so we won't need it on next release... but if the issue is still not fixed, then we will need to send this same PR to it

fruch commented:
> > @soyacz why sending this to master ? (and then we'll backport, like any other issue ?)
> 
> this must be decided, but this is only a workaround for an issue with the manager... so, hopefully it will be fixed soon, so we won't need it on next release... but if the issue is still not fixed, then we will need to send this same PR to it

still changes should go via master, only backport PRs might go directly to branches.

soyacz commented:
> @soyacz why sending this to master ? (and then we'll backport, like any other issue ?)

I needed to execute a test for 2022.2 - so I did it from this branch.
I rebased, switched base branch and added backport labels.

fgelcer commented:
> > @soyacz why sending this to master ? (and then we'll backport, like any other issue ?)
> 
> I needed to execute a test for 2022.2 - so I did it from this branch. I rebased, switched base branch and added backport labels.

@roydahan , shall we have a task to revert this change, once the manager issue is fixed?

fruch reviewed (approved):
LGTM

roydahan commented:
> > > @soyacz why sending this to master ? (and then we'll backport, like any other issue ?)
> > 
> > 
> > I needed to execute a test for 2022.2 - so I did it from this branch. I rebased, switched base branch and added backport labels.
> 
> @roydahan , shall we have a task to revert this change, once the manager issue is fixed?

No, the manager should have a scale test as a reproducer.

roydahan commented:
@soyacz you can backport.
